bpf/lib/nodeport.h

```diff
@@ -311,10 +316,14 @@ static __always_inline int xlate_dsr_v6(struct __ctx_buff *ctx,
 __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV6_NODEPORT_DSR)
 int tail_nodeport_ipv6_dsr(struct __ctx_buff *ctx)
 {
+	struct bpf_fib_lookup fib_params = {
+		.family		= AF_INET6,
+		.ifindex	= DIRECT_ROUTING_DEV_IFINDEX,
+	};
```

Comment:
We probably don't need to initialize this until the `fib_lookup()` fallback case below.

Reply:
At least the ifindex is used also in the non-fib case. Given prior case was unconditional `struct bpf_fib_lookup fib_params = {};` I thought I'd reuse it for the neigh map case. I can look into reworking it a bit as follow-up.